pkg/api/readyapi/readyapi.go

```diff
@@ -9,3 +9,9 @@ type MsgTETRAGONReady struct{}
 func (msg *MsgTETRAGONReady) HandleMessage() *tetragon.GetEventsResponse {
 	return nil
 }
+
+func (msg *MsgTETRAGONReady) SetCached() {}
```

Comment:
Maybe a comment here to explain that MessageReady.IsCached() should always be true and that's why we're leaving this blank?